index.js

```diff
@@ -20,10 +20,17 @@ class PinoTransform extends Transform {
   }
 
   _transform (chunk, encoding, callback) {
-    const { column, passThrough } = this.opts
+    const { column, passThrough, wrapNonJson } = this.opts
     const content = chunk.toString('utf-8')
 
-    buffer.push({ [column]: content })
+    // Assumes that JSON written by PINO is contained within {"...}.
+    if (wrapNonJson && (! content.startsWith('{"') || ! content.endsWith('}'))) {
```

Comment:
There can be cases where the message starts and ends with a brace, but is still invalid json.
Could we use JSON.parse here?

Reply:
Hi! I considered using `JSON.parse` at first. However, would there be a use case for strict json checking? Or could we use a simpler criteria/heuristic for performance? 
I am considering that Pino will always produce well-behaved messages. And being very unlikely some other module writing a line with invalid JSON. Or a text message that starts with {“ and ends with ” by coincidence.
Further, I saw in history that JSON parse was removed recently from pino-postgres and I was trying to avoid reintroducing it again.
But I would not object using  `JSON.parse` if you prefer.

Reply:
It's hard to predict what people are putting in their log messages. We can't exclude the possibility that there is someone who's going to do `console.log('{ something }')`. I'd prefer correctness over raw performance in this type of situation.

I did have the `JSON.parse` before, and I'm not exactly sure why I had removed it, possibly because I was tailoring usage based on my specific use case, where I expected all messages to be properly formatted. Now that you brought this issue up, I do think proper payload validation should be in place, in which case `JSON.parse` seems like a good way to ensure validity.

Reply:
I just looked at pino-pretty. It also uses `JSON.parse` to decide whether the message is from Pino or not.
I also searched for alternatives to `JSON.parse` that just test the string without creating an object structure. I only found complex regular expressions. However, I do not feel confortable copying someone else's REs without having a clear understanding of them. And I am not sure if a complex RE would perform better than `JSON.parse`. (And I remember from Computer Science classes that RE are not suitable for such purpose.)

At least for now, `JSON.parse` seems to be the best way to keep it simple and safe.

Reply:
I think I do not know how to update this PR with the new code... :-(